client/app/components/ServicesTemplate/ServicesCardContent/styles.scss

```diff
@@ -11,6 +11,8 @@
 
 .title {
   padding-bottom: 0.25rem;
+  // This padding-top doesn't work. 
+  padding-top: .70rem !important; 
 }
```

Comment:
please avoid committing !important, and/or code that doesn't appear to be working